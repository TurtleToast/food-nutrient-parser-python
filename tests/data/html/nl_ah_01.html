<h2>Voedingswaarden</h2>
<p>Deze waarden gelden voor het onbereide product.</p>
<table class="product-info-nutrition_table__12abc">
    <thead>
        <tr>
            <th scope="col">Soort</th>
            <th scope="col">Per 100 Gram</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td class="">Energie</td>
            <td>1141 kJ (273 kcal)</td>
        </tr>
        <tr>
            <td class="">Vet</td>
            <td>10 g</td>
        </tr>
        <tr>
            <td>waarvan verzadigd</td>
            <td>3.6 g</td>
        </tr>
        <tr>
            <td class="">Koolhydraten</td>
            <td>25 g</td>
        </tr>
        <tr>
            <td>waarvan suikers</td>
            <td>2.3 g</td>
        </tr>
        <tr>
            <td class="">Eiwitten</td>
            <td>11 g</td>
        </tr>
        <tr>
            <td class="">Zout</td>
            <td>0.9 g</td>
        </tr>
    </tbody>
</table>
</div>