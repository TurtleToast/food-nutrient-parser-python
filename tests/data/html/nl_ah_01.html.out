{
  'columns': [
    {
      'rows': [
        { 'name': 'Energie', 'amount': '1141 kJ' },
        { 'name': 'Energie', 'amount': '273 kcal' },
        { 'name': 'Vet', 'amount': '10 g' },
        { 'name': 'waarvan verzadigd', 'amount': '3.6 g' },
        { 'name': 'Koolhydraten', 'amount': '25 g' },
        { 'name': 'waarvan suikers', 'amount': '2.3 g' },
        { 'name': 'Eiwitten', 'amount': '11 g' },
        { 'name': 'Zout', 'amount': '0.9 g' },
      ]
    }
  ]
}